src/Mod/Part/App/ParabolaPy.xml

```diff
@@ -17,8 +17,9 @@
         </Documentation>
         <Methode Name="compute">
             <Documentation>
-                <UserDocu>compute(p1,p2,p3)
-                    The three points must lie on a plane parallel to xy plane and must not be collinear</UserDocu>
+                <UserDocu>compute(p1,p2,p3) -> None
```

Comment:
Sometimes you write ‘->’ and sometimes’-&gt;’ would be good if you can choose just one. Is ‘>’ even valid in xml?

Reply:
True, (resolved)